src/interfaces/CommandInt.ts

```diff
@@ -0,0 +1,9 @@
+import { Message } from "discord.js";
+import { BotConfigInt } from "./BotConfigInt";
+
+export interface ComamndInt {
```

Comment:
I think I might better to append with the full word `Interface` instead of the abbreviation `Int`

Reply:
Fixed in commit `eca1b2a`.